ui/client/http/HttpService.js

```diff
@@ -267,10 +266,9 @@ export default {
   },
 
   compareProcesses(processId, thisVersion, otherVersion, businessView, remoteEnv) {
-    const queryParams = this.businessViewQueryParams(businessView)
     const path = remoteEnv ? "remoteEnvironment" : "processes"
 
-    return api.get(`/${path}/${processId}/${thisVersion}/compare/${otherVersion}`, {params: queryParams})
+    return api.get(`/${path}/${processId}/${thisVersion}/compare/${otherVersion}`, {params: {businessView}})
```

Comment:
can we get rid of this method altogether (businessViewQueryParams)?

Reply:
Yes, we can! :)

Reply:
Done.